docs/kura-wires/assets-as-wire-components.md

```diff
@@ -114,7 +114,6 @@ The WireAsset component provides the following global (non per-channel) configur
 * **emit.all.channels**: Specifies whether the values of all READ or READ_WRITE channels should be emitted when a channel event is received in listen mode. If set to true, the values for all channels will be read and emitted, if set to false, only the value for the channel related to the event will be emitted.
 
 * **timestamp.mode**: Allows to configure how timestamps are emitted, the following modes are supported:
-
   * **NO_TIMESTAMPS**: no timestamp-related properties will be emitted.
```

Comment:
It seems that 4 spaces are needed for the indentation. At least, it works for me...